tests/foreman/cli/test_repository.py

```diff
@@ -361,8 +522,15 @@ def test_positive_update_url(self):
         @Assert: Repository url is updated
         """
         new_repo = self._make_repository()
-        for url in (FAKE_4_YUM_REPO, FAKE_1_PUPPET_REPO, FAKE_2_PUPPET_REPO,
-                    FAKE_3_PUPPET_REPO, FAKE_2_YUM_REPO):
+        # generate a tuple of auth repos with all valid credentials
+        auth_repos = [
+            repo.format(creds['login'], creds['pass'])
+            for creds in valid_http_credentials(url_encoded=True)
+            for repo in (FAKE_5_YUM_REPO, FAKE_7_PUPPET_REPO)
+        ]
+
+        for url in [FAKE_4_YUM_REPO, FAKE_1_PUPPET_REPO, FAKE_2_PUPPET_REPO,
```

Comment:
We had conversation regarding that case and you left it as it was. I still don't like the idea, but, probably, impact is minimal, so let's leave it, but please, in future, let's get to some agreement, not simply ignore comments.